main/templates/main/help.html

```diff
@@ -15,15 +15,15 @@
 
 
 {% block content %}
-<div class="uu-inner-container">
+<div class="uu-container">
     <div class="col-12">
         <h1>
             {% trans 'main:help:header' %}
         </h1>
```

Comment:
Help! I need a hero!

Sorry... 

Anyway, a hero might be nice, but not required in my eyes